Scripts/lint.sh

```
@@ -1,45 +1,101 @@
-	SWIFTFORMAT_OPTIONS=""
+elif [ "$LINT_MODE" = "STRICT" ]; then
+	SWIFTFORMAT_LINT_OPTIONS="--strict"
 	SWIFTLINT_OPTIONS="--strict"
-else 
-	SWIFTFORMAT_OPTIONS=""
+else
+	SWIFTFORMAT_LINT_OPTIONS=""
 	SWIFTLINT_OPTIONS=""
 fi
 
 pushd $PACKAGE_DIR
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)